packages/dapp-simple-exchange/contract/deploy.js

```diff
@@ -65,52 +58,8 @@ export default async function deployContract(homeP, { bundleSource, pathResolve
   // 3. Get the instanceHandle
 
   const {
-    extent: { instanceHandle },
-  } = await invite~.getBalance();
-
-  // =====================
-  // === AWAITING TURN ===
-  // =====================
-
-  // 4. Get the contract terms and assays
-
-  const { terms: { assays }} = await homeP~.zoe~.getInstance(instanceHandle);
-
-  // =====================
-  // === AWAITING TURN ===
-  // =====================
-
-  // 5. Offer rules
-  const [unit0, unit1] = await Promise.all([
```

Comment:
This code was creating the initial liquidity. Are we doing that elsewhere?

Reply:
Does simple-exchange need liquidity?  I didn't know that.

Reply:
It could benefit from some initial orders in the order book in the same way that we added liquidity to autoswap. It will still work without seeding with initial orders, but will be a worse user experience.

Reply:
That's a good idea.  I'll make this work.